packages/relay/src/lib/eth.ts

```diff
@@ -1150,11 +1190,19 @@ export class EthImpl implements Eth {
     const requestIdPrefix = formatRequestIdMessage(requestId);
     this.logger.trace(`${requestIdPrefix} call(hash=${JSON.stringify(call)}, blockParam=${blockParam})`, call, blockParam);
 
+    const currentTimestamp: number = Date.now();
```

Comment:
Q: WHat's the timestamp label for?

Reply:
Removed.  I am not sure what I was thinking.